python/test_file_generator.py

```diff
@@ -813,17 +809,6 @@ def v633_1_file(self):
 		print("writing {name}".format(name=name))
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
-	def v633_2_file(self):
```

Comment:
Is this file not being used by the platform?

Reply:
Yes, it's just v633.

Reply:
Yes, it's just a v633 edit.